packages/mobile/src/analytics/Events.tsx

```diff
@@ -353,6 +354,8 @@ export enum FiatExchangeEvents {
   cash_in_success = 'cash_in_success',
 
   cico_add_funds_selected = 'cico_add_funds_selected',
+  // Add fund flow entered through home screen cash in bottom sheet
+  cico_add_funds_selected_home = 'cico_add_funds_selected_home',
```

Comment:
Nit: I'd rename this to something like `cico_add_funds_bottom_sheet_selected` or something so that it's logically grouped with the impression event